### DISCUSSION TITLE
What service does the package use for playback?

### DISCUSSION THREAD

**Discussion by TechPro424** [Androz2091/discord-player#793] [General]
What service does the package use for playback?
Asking because, if it uses YouTube for playback, then using this package will probably lead to the bot's shutdown/removal of music commands, similar to Groovy and Rythm.

**Comment by twlite** [comment]
Discord Player by default supports youtube and soundcloud, but you can use extractors for more.

**Reply by TechPro424** [reply]
So, it takes direct playback from soundcloud?
and if I use an extractor for soptify, it takes direct playback from spotify?
